react-native/ios/RealmReact.xcodeproj/project.pbxproj

```diff
@@ -345,7 +345,7 @@
 					"$(SRCROOT)/../../tests/react-test-app/node_modules/react-native/React/**",
 					"$(SRCROOT)/../../examples/ReactExample/node_modules/react-native/React/**",
 				);
-				OTHER_LIBTOOLFLAGS = "$(SRCROOT)/../../core/librealm-ios-bitcode.a $(BUILT_PRODUCTS_DIR)/libRealmJS.a $(BUILT_PRODUCTS_DIR)/libGCDWebServers.a";
+				OTHER_LIBTOOLFLAGS = "$(BUILT_PRODUCTS_DIR)/libRealmJS.a $(BUILT_PRODUCTS_DIR)/libGCDWebServers.a";
```

Comment:
This is. Core is now linked into `libRealmJS` so that all users of that library don't also need to include it.